src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,157 @@
+footer {
+  font-size: 0.875rem;
+  background: $gray-dark;
+  color: transparentize($white,0.5);
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.footer-links {
+  text-align: center;
+  list-style-type: none;
+  padding-left: 0;
+  margin-bottom: 0;
+
```

Comment:
Um we can't anticipate what and how many footer links are gonna be there. With flexbox we can wrap links into multilines but we can't have space around each links equally set. 

```
  [ item 1 ]     [ item 2 item 2 item 2 ]    [ item 3 ]
    [ item 4 item 4 item 4 item 4 ] [ item 5 item 5 ] 
[ item 6  ] [ item 7 ] [ item 8 item 8 item 8 ] [ item 9 ]

```